### ISSUE TITLE
Bloom Not Seeing Database

### ISSUE THREAD

**Issue by andyhegedus** [neo4j-apps/neo4j-bloom#57]
I am having recent connection issues.
Database 4.4.8 which was previously working fine
Open in Bloom and perspective is empty and no option to generate one.
Also Noticing trouble connecting via Desktop Browser.  Open the database via open button and it does not connect.
I need to use the connect server button and it is using port 11003 instead of the normal 7687 port.
Also if I close the browser and reopen I also have to reconnect via different port connection.

I think this also may have occurred when the python driver was upgraded to version 5.

Version | 4.4.8
-- | --
Edition | enterprise
Status | Active
IP address | localhost
Bolt port | 11003
HTTP port | 11004
HTTPS port | 7473

Andy

**Comment by angeliquekom** [comment]
hi, it looks like a connection issue, but we are investigating it. Meanwhile, could you check your logs if an error message indicates any issues on binding to the port?

**Comment by palmergw** [comment]
I am also running into this running a local instance of 5.8 using ports 7687 and 7474.
I had just installed a new version of the Bloom app.

I tried forcing a reinstall of Bloom, but that didn't change anything.
Sometimes Bloom will load the default database no problem, and usually if I reload Bloom it will tell me that there is no active DB. I can't find errors on the Neo4j Desktop side. When it does load, under perspectives it behaves normally for the default DB, but I cannot switch to my other DB.

I also spun up a new 5.8 DBMS to test there, same results.

Message in Bloom after a reload:

> We couldn't find an active graph in Neo4j Desktop. Please check that you have connected to a graph.
